extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.icons.js

```diff
@@ -0,0 +1,60 @@
+			// wds-icons-volume-off-small.svg
+			volumeOff: '<path d="M8.45 2.17L4.664 6.28H1.036C.256 6.28 0 6.739 0 7.175v3.522c0 .436.256.985 1.036.985h3.646l3.785 4.176a1.1 1.1 0 0 0 .533.143.964.964 0 0 0 .5-.137c.33-.185.5-.526.5-.897V3.013c0-.37-.17-.713-.5-.898-.33-.186-.72-.13-1.05.054zm7.192 7.33l2.121-2.122a.807.807 0 1 0-1.142-1.141l-2.122 2.12-2.12-2.12a.808.808 0 0 0-1.142 1.141L13.358 9.5l-2.121 2.121a.807.807 0 1 0 1.142 1.142l2.12-2.12 2.122 2.12a.805.805 0 0 0 1.142 0 .807.807 0 0 0 0-1.142L15.642 9.5z" fill-rule="evenodd"/>\n',
+			// wds-icons-volume-small.svg
+			volumeOn: '<g fill-rule="evenodd"><path d="M8.45 2.17L4.664 6.28H1.036C.256 6.28 0 6.739 0 7.175v3.522c0 .436.256.985 1.036.985h3.646l3.785 4.176a1.1 1.1 0 0 0 .533.143.964.964 0 0 0 .5-.137c.33-.185.5-.526.5-.897V3.013c0-.37-.17-.713-.5-.898-.33-.186-.72-.13-1.05.054zm4.95 10.156a4.393 4.393 0 0 0 0-6.19.708.708 0 0 0-1.004 1 2.978 2.978 0 0 1 0 4.192.707.707 0 1 0 1.003.998z"/><path d="M17.515 9.231A6.186 6.186 0 0 0 15.7 4.84a.707.707 0 1 0-1.003.998A4.777 4.777 0 0 1 16.1 9.231a4.778 4.778 0 0 1-1.4 3.394.708.708 0 1 0 1.002.999 6.186 6.186 0 0 0 1.814-4.393z"/></g>'
+		};
+
+		/**
+		 * Replaces inner markup of JW SVG's with SVG from Design System
+		 * We cannot override the whole SVG as this makes the player misbehave
+		 * (e.g.) show fullscreen icon twice, next to each other
+		 *
+		 * @param {HTMLElement} icon
+		 * @param {string} iconHtml
+		 */
+		function replaceJWIconWithCustom(icon, iconHtml) {
+			icon.innerHTML = iconHtml;
+
+			// JW icons have viewBox=0,0,240,240 so we need to override it,
+			// otherwise our icons would be too small
+			icon.setAttribute('viewBox', '0 0 24 24');
+		}
+
+		/**
+		 * @param {HTMLElement} videoPlayerElement
+		 */
+		function replaceIcons(videoPlayerElement) {
+			var controlBar = videoPlayerElement.querySelector('.jw-controlbar');
+
+			replaceJWIconWithCustom(controlBar.querySelector('.jw-svg-icon-play'), icons.play);
```

Comment:
```
[...classes].forEach((0 => ...)
```

Reply:
If they were to do that they would also have to extract class names or convert keys of the icons object to `this-format` rather than `thisFormat`.

Reply:
@KockaAdmiralac, why?

i thought about sth like:
```js
[
    { selector: '.jw-svg-icon-play', icon: icons.play },
    { selector: '.jw-svg-icon-pause', icon: icons.pause },
    ...
].forEach(function (obj) {
    replaceJWIconWithCustom(controlBar.querySelector(obj.selector), obj.icon)
})
```